test/lib/mayaUsd/render/mayaToHydra/testStageVariants.py

```diff
@@ -0,0 +1,119 @@
+#
+# Copyright 2024 Autodesk, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import maya.cmds as cmds
+import fixturesUtils
+import mtohUtils
+import unittest
+import testUtils
+import mayaUtils
+import usdUtils
+import ufe
+
+class TestStageVariants(mtohUtils.MtohTestCase): #Subclassing mtohUtils.MtohTestCase to be able to call self.assertSnapshotClose
+    # MayaHydraBaseTestCase.setUpClass requirement.
+    _file = __file__
+
+    IMAGE_DIFF_FAIL_THRESHOLD = 0.1
+    IMAGE_DIFF_FAIL_PERCENT = 0.3
+
+    @unittest.skipUnless(mtohUtils.checkForMayaUsdPlugin(), "Requires Maya USD Plugin.")
+    def test_UsdStageVariants(self):
+        from mayaUsd import lib as mayaUsdLib
+        cmds.file(new=True, force=True)
+
+        testFile = mayaUtils.openTestScene(
+                "testStageVariants",
+                "testStageVariants.ma")
+        self.setHdStormRenderer()
+        cmds.refresh()
+
+        shapeNode = "variantsShape"
+        shapeStage = mayaUsdLib.GetPrim(shapeNode).GetStage()
+        
+        #Check geometry variant
+        rootPrim = shapeStage.GetPrimAtPath('/Cubes')
+        self.assertFalse(rootPrim== None)
```

Comment:
Minor: better to use assertIsNotNone() when testing against None, provides clearer output on failure:
https://codereview.doctor/features/python/best-practice/use-assert-is-not-none